application/src/main/resources/thingsboard.yml

```diff
@@ -166,6 +166,7 @@ cassandra:
     ts_key_value_partitioning: "${TS_KV_PARTITIONING:MONTHS}"
     ts_key_value_ttl: "${TS_KV_TTL:0}"
     events_ttl: "${TS_EVENTS_TTL:0}"
+    debug_events_ttl: "${DEBUG_EVENTS_TTL:WEEK}"
```

Comment:
This should be long value, similar to ts_key_value_ttl or events_ttl. I believe Cassandra TTL is set in seconds, but I may be wrong.